ShtefanIlya/HW6/hw_6.1.py

```diff
@@ -0,0 +1,32 @@
+l = []
+while True:
+    element = input("Enter an element: ")
+    if element == "stop" or element == "STOP":
+        break
+    elif element.lstrip('-').isdigit():
+        l.append(int(element))
+    else:
+        print("""If you wish to stop the loop,
+just input \"stop\" or \"STOP\"""")
+
+l_div_for_2 = []
+l_div_for_3 = []
+l_not_two_three = []
+aqq = []
+print(f"List: {l}\n")
+
```

Comment:
good idea
look at this version of code 
l1 = [ x for x in range(1, 11) if x % 2 == 0 ]
l2 = [ x for x in range(1, 11) if x % 2 == 1 and x % 3 == 0 ]
l3 = [ x for x in range(1, 11) if not x % 2 == 0 and not x % 3 == 0 ]
print(l1, 'is even multiple of 2')
print(l2, 'is an odd multiple of 3')
print(l3, 'not divisible by 2 and 3')